content/blog/2020-04-dplyr-1-0-0-colwise/index.Rmarkdown

```diff
@@ -0,0 +1,173 @@
+---
+title: 'dplyr 1.0.0: working across columns'
+author: Hadley Wickham
+date: '2020-04-03'
+slug: dplyr-1-0-0-colwise
+categories:
+  - package
+tags:
+  - dplyr
+photo:
+  author: Alexey Derevtsov
+  url: https://unsplash.com/photos/Atl1BCVM2fo
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+This post is the latest in a series of post leading up the the dplyr 1.0.0 release. So far, the series has covered:
+
+* [Major lifecycle changes](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-is-coming-soon/).
+* [New `summarise()` features](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-summarise/).
+* [`select()`, `rename()`, `relocate()`](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-select-rename-relocate/).
+
+Today, I wanted to talk a little bit about the new `across()` function that makes it easy to perform the same operation to multiple columns.
```

Comment:
```suggestion
Today, I wanted to talk a little bit about the new `across()` function that makes it easy to perform the same operation on multiple columns.
```

Reply:
I think [perform takes on](https://inspirassion.com/en/prep/perform) as its preposition here, though you could switch "perform" to _do_ and then keep "to" instead of _on_.